service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/SubmittedCaseEventHandlerTest.java

```diff
@@ -130,7 +130,7 @@ void init() {
         @ParameterizedTest
         @EnumSource(value = State.class, mode = EXCLUDE, names = {"OPEN"})
         void shouldNotPayIfCaseStateIsDifferentThanOpen(State state) {
-            final CallbackRequest request = callbackRequest(state);
+            final CallbackRequest request = callbackRequest();
```

Comment:
The state from this parameterised test is no longer being used

Reply:
although why did this test pass then 🤔 
the state would have been open so going by the name of the test it should fail

Reply:
ahh it seems to be due to the data passed to the event. If you alter the data map in the callback request to contain this entry `Map.of("displayAmountToPay", "Yes")` then the tests will fail.

Reply:
As it is now it is hitting this logic
```java
        switch (getPaymentDecision(caseDetails)) {
            case YES:
                makePaymentForCaseOrders(event);
                return;
            case NO:
                handlePaymentNotTaken(event);
                return;
            default:
                log.info("Payment not taken for case {}.", event.getCallbackRequest().getCaseDetails().getId());
        }
```
and as the payment decision wasn't set it hit the default case